helpers/middleman/partners.rb

```diff
@@ -38,6 +40,10 @@ def categories_collection(lang = default_locale_obj[:lang])
 
   # localize_value should always have a third argument: fallback_locale!
   def partners_categories(lang = default_locale_obj[:lang], withIndex: false)
+    @partners_categories ||= {}
+    @partners_categories[lang] ||= {}
+    return @partners_categories[lang][withIndex] if @partners_categories[lang][withIndex]
```

Comment:
See above (The Ruby Way ™).

Reply:
How can I do `@partners_categories[lang][withIndex] ||= ` if `@partners_categories` is `nil`?

Also, is `@partners_categories[nil]` or `@partners_categories[false]` a valid key index?

Reply:
You cannot. You always have to `@categories_collection ||= {}` as a first step.